tests/unit/data/test_particle_collection.py

```diff
@@ -11,7 +11,7 @@ def test_find(particle_database):
     assert f2_1950.mass == 1.936
     phi = particle_database.find("phi(1020)")
     assert phi.pid == 333
-    assert phi.width == 0.004249
+    assert phi.width == 0.004248999999999999
```

Comment:
Have a look at https://docs.pytest.org/en/latest/reference.html#pytest-approx

Reply:
I wonder how that would work for a class with several attributes. I did it with these strings, because I assume (#33) these particle lists will disappear eventually anyway. At first I also rounded the floats upon imports, and even tried out `Decimal`, but all of those introduce hidden behaviour.

Reply:
See 170a905da2d38760365d4d2188b039ddaee87a19